composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "composer/installers": "*",
         "silverstripe/framework": "~3.1",
```

Comment:
Could you please also update the framework requirement to `^3.7`?
`SS_Object` isn't a thing in earlier versions :)

Reply:
True, missed that since my project had 3.7 as a req.